openapi-issuer.yaml

```
@@ -108,6 +108,31 @@ paths:
           $ref: "#/components/responses/NotFound"
         default:
           $ref: "#/components/responses/Unexpected"
+    patch:
+      operationId: patchSignatureRequestStatus
```

Try not to name things leaking implementation details that are subject to changes ('patch').
```suggestion
      operationId: setSignatureRequestStatus
```